static/styles/compose.scss

```diff
@@ -349,6 +349,23 @@ input.recipient_box {
     width: 100%;
 }
 
+#compose-send-button {
+    border-radius: 4px;
```

Comment:
Maybe worth adding a comment that this floats right in order to ensure the desired tab ordering.

Reply:
The `float: right` does not change the tab ordering, just the visual position.

Reply:
I'll add what I have in mind.